easybuild/easyconfigs/y/Yasm/Yasm-1.3.0-goolf-1.7.20.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'Yasm'
+version = '1.3.0'
+#versionsuffix = '-20141219'
+
+description = "Yasm-1.3.0: Complete rewrite of the NASM assembler with BSD license"
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://www.tortall.net/projects/yasm/releases/']
+
+homepage = 'http://www.tortall.net/projects/yasm/'
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+sanity_check_paths = {
+    'files': [],   ## fixme
+    'dirs': ['.'], ## fixme
+}
+
+parallel = 1         # this is a very conservative choice ## fixme
+moduleclass = 'base' # please change this from base to right category ## fixme
```

Comment:
@eediaz1987 same here, please pick an appropriate module class, and verify whether `parallel = 1` is really needed (if not, remove it)